app/hooks/handle_send_message.ts

```diff
@@ -119,12 +121,22 @@ export const useHandleSendMessage = ({
             response = await createPost(serverUrl, post, postFiles);
         }
 
+        if (response.error) {
+            showSnackBar({
+                barType: SNACK_BAR_TYPE.CREATE_POST_ERROR,
+                customMessage: getErrorMessage(response.error),
+                type: 'error',
+                keepOpen: true,
```

Comment:
what is the reason to keep it open?

Reply:
Discussed this with @asaadmahmood . No need to keep it once always. Will remove it.